build.fsx

```diff
@@ -128,6 +128,13 @@ Target "CleanDocs" (fun _ ->
     CleanDirs ["docs/output"]
 )
 
+#load "paket-files/build/matthid/FAKE/src/app/Fake.DotNet.Cli/Dotnet.fs"
+open Fake.DotNet.Cli
+
+Target "InstallDotnetCore" (fun _ ->
+    DotnetCliInstall Preview2ToolingOptions
```

Comment:
Credit goes to @dolly22 https://github.com/dolly22/fake.dotnet 
I extended it to work with the bash installer on linux/mac